src/java/org/apache/cassandra/streaming/StreamingState.java

```diff
@@ -40,7 +41,6 @@
 import org.apache.cassandra.utils.Clock;
 import org.apache.cassandra.utils.ObjectSizes;
 import org.apache.cassandra.utils.TimeUUID;
-import org.checkerframework.checker.nullness.qual.Nullable;
```

Comment:
Is it worth adding this class to the `IllegalImport` checkstyle rule?

Reply:
no, because it will not be on the class path to import anymore